### DISCUSSION TITLE
github pages

### DISCUSSION THREAD

**Discussion by Aidar1983** [flet-dev/flet#2887] [Q&A]
### Question

Добрый день! Есть ли где то пошаговая инструкция по компиляции приложения, на самом простом примере для размещения на github pages ? Я делал все по инструкции размещенная на официальном сайте(https://flat.dev/docs/guides/python/packaging-app-for-distribution/)  через flet build web и у меня не получилось. Вот мой репозиторий с примером https://github.com/Aidar1983/flet-test, при запуске его на  github pages открывается пустая страничка. 

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
You need to set the `--base-url` when packaging.
Docs: https://flet.dev/docs/guides/python/packaging-app-for-distribution/#hosting-website-in-a-sub-directory
